packages/form-js-viewer/src/core/Validator.js

```diff
@@ -54,6 +58,20 @@ export default class Validator {
       ];
     }
 
+    if ('validationType' in validate && value && validate.validationType === 'phone' && !PHONE_PATTERN.test(value)) {
+      errors = [
+        ...errors,
+        'Field must be a valid phone number.'
```

Comment:
Should maybe give a hint in the error message about what a valid phone number looks like? Might be ok for the demo though.

![image](https://user-images.githubusercontent.com/9433996/201030957-ecef05b7-8a2c-4597-9245-7ad844c651d8.png)